src/app/Component/chatroom/ChatRoom.js

```diff
@@ -57,13 +65,16 @@ class ChatRoom extends Component {
     if(msgIds && msgIds.length){
       this.socket.emit("read_status", { username: this.props.user.username, client2: this.props.client.username, messageIds: msgIds });
     }
+    console.log(data.messages);
```

Comment:
remove console logs if it is not used.Only console logs should be there in api response